src/analyses/goto_rw.cpp

```diff
@@ -524,6 +525,13 @@ void rw_range_sett::get_objects_rec(
     get_objects_array(mode, to_array_expr(expr), range_start, size);
   else if(expr.id()==ID_struct)
     get_objects_struct(mode, to_struct_expr(expr), range_start, size);
+  else if(expr.id()==ID_dynamic_object)
+  {
+    // Dynamic object expressions should always have two operands.
+    PRECONDITION(expr.operands().size() == 2);
+    const auto obj_instance = to_dynamic_object_expr(expr).get_instance();
+    add(mode, "dynamic_object-" + std::to_string(obj_instance), 0, -1);
```

Comment:
Any risk of name clash? Most analyses seem to namespace dynamic objects to avoid this